src/AutoRest.CSharp/AutoRest/Plugins/CSharpProj.cs

```diff
@@ -22,15 +24,53 @@ internal class CSharpProj : IPlugin
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <Nullable>annotations</Nullable>
   </PropertyGroup>
-
+{0}
   <ItemGroup>
-    <PackageReference Include=""Azure.Core"" Version=""1.1.0"" />
-    <PackageReference Include=""System.Text.Json"" Version=""4.6.0"" />
+    <PackageReference Include=""Azure.Core"" Version=""1.6.0"" />
   </ItemGroup>
 
 </Project>
+";
+         private string _csProjPackageReference = @"
+  <PropertyGroup>
+    <LangVersion>8.0</LangVersion>
+    <IncludeGeneratorSharedCode>true</IncludeGeneratorSharedCode>
+	<RestoreAdditionalProjectSources>https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json</RestoreAdditionalProjectSources>
+  </PropertyGroup>
 
+  <ItemGroup>
+	<PackageReference Include=""Microsoft.Azure.AutoRest.CSharp"" Version=""{0}"" />
```

Comment:
nit spaces

Reply:
Fixed it. 

It was weird. In VS it was looking good but I see no spaces in VS code same as this.